apps/storefront/lib/auth/useApolloClient.ts

```diff
@@ -0,0 +1,31 @@
+import { Fetch } from "@/lib/auth/types";
+import { API_URI } from "@/lib/const";
+import { ApolloClient, createHttpLink, InMemoryCache } from "@apollo/client";
+import { useMemo } from "react";
+
+import { typePolicies } from "./typePolicies";
+
+// for static geenration of pages, we don't need auth there
+export const staticApolloClient = new ApolloClient({
```

Comment:
Why is it called `static`? Since its purpose is to make unauthenticated requests from backend, maybe we should call it `severApolloClient` or `unauthenticatedAppolloClient`?

Reply:
Indeed, serverApolloClient sounds better. Didn't really figure out a great name here so sure 😅